### PR TITLE
Testing/lyrics card

### PR DESCRIPTION
**Pull request by elizhahn**
## What does this PR do (summary of changes)?
 - This adds end to end testing for the new lyric feature
 - Testing attributes were added to the appropriate elements
 - Lyric card was tested for both my songs and song book views
 - intercepts were updated to use deployed server url
 - Song book and my songs were updated to check for the microphone icon, and the test was updated to also check for the new home element instead of the word title
## How should it be tested?
 -  The lyrics testing should be passing, there are 5 tests failing because of updated functionality that hasn't been tested yet/changed
## Future Iterations:
 - Next branch will address the navigation tests, search bar messages, server messages

### CODE REVIEW

**Comment by m1073496** [comment]
[Add testing for lyrics](https://trello.com/c/FagBRliM/101-add-testing-for-lyrics)

File: cypress/integration/carry_okay_spec.js [hunk 1]
```diff
@@ -39,13 +39,13 @@ describe('CarryOkay', () => {
...
       cy.get('[data-cy=my-songs-nav]').click();
-      cy.expect(true).to.equal(true);
+      cy.expect(true).to.equal(false);
     });
     
     //still missing this functionality
     it('should be able to navigate from Song Book view to Home view', () => {
       cy.get('[data-cy=song-book-nav]').click();
-      cy.expect(true).to.equal(true);
+      cy.expect(true).to.equal(false);
```
Comment (m1073496) [inline cypress/integration/carry_okay_spec.js:48]: We'll tackle these today!

File: cypress/integration/carry_okay_spec.js [hunk 2]
```diff
@@ -105,20 +106,27 @@ describe('CarryOkay', () => {
...
     cy.get('[data-cy=song-card]').eq(0).contains("Radiohead");
     cy.get('[data-cy=song-card]').eq(1).contains("Usher");
+    cy.get('[data-cy=song-card]').first().should("contain", "Jigsaw falling into place")
+    .and("contain", "Radiohead")
+    .and("contain","Electronica")
+    .and("contain","Rock")
+  cy.get('[data-cy=album-img]').first().should('exist');
+  cy.get('[data-cy=song-card-btn]').first().should('exist');
+  cy.get('[data-cy=microphone-icon]').should('exist');
   });
```
Comment (m1073496) [inline cypress/integration/carry_okay_spec.js:130]: I wonder if we should create a map and loop through each song

**Review by m1073496 — APPROVED** [review]

File: cypress/integration/carry_okay_spec.js (hunk 2 above)
Comment (elizhahn) [reply cypress/integration/carry_okay_spec.js:130]: ok yeah, I can implement that

**Pull request merged**